hubblestack/files/hubblestack_nova/vulners_scanner.py

```diff
@@ -68,7 +68,26 @@ def _get_local_packages():
     '''
 
     local_packages = __salt__['pkg.list_pkgs']()
-    return ['{0}-{1}'.format(pkg, local_packages[pkg]) for pkg in local_packages]
```

Comment:
The existing implementation assumes packages are in RedHat format only. The new implementation uses salt grains to determine the `os_family` before deciding how packages should be formatted.